gravitee-apim-rest-api/gravitee-apim-rest-api-services/gravitee-apim-rest-api-services-dynamic-properties/src/main/java/io/gravitee/rest/api/services/dynamicproperties/DynamicPropertyUpdater.java

```diff
@@ -64,12 +68,16 @@ private void authenticateAsAdmin() {
 
     @Override
     public void handle(Long event) {
+        handle();
+    }
+
+    protected CompletableFuture<Collection<DynamicProperty>> handle() {
```

Comment:
the returned value is not used. Can we make without it ?

Reply:
> the returned value is not used. Can we make without it ?

It's usefull for test